src/lib/viewers/archive/__tests__/Breadcrumbs-test-react.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import Breadcrumbs from '../Breadcrumbs';
-import { ROOT_FOLDER, VIEWS } from '../constants';
+import { VIEWS } from '../constants';
 
 describe('lib/viewers/archive/Breadcrumbs', () => {
-    const getComponent = props => shallow(<Breadcrumbs {...props} />);
+    const renderView = props => render(<Breadcrumbs {...props} />);
```

Comment:
Nit: It's a small and simple render, so I think using `render` directly would be more readable. I highly recommend not using helper functions like that whenever possible or trying to name the helper function in a more meaningful manner, i.e. `renderBreadcrumb`. `renderView` is too generic IMHO, and because of that I need to check the helper implementation in order to understand the test itself.